docs/contributors/code/release.md

```diff
@@ -50,22 +50,44 @@ Once released, all that's left to do is writing a release post on [make.wordpres
 -   Remove mobile app pull request entries.
 
-Once you have cleaned up the changelog, choose a few features to highlight in the release post and record an animation of them in use.
+#### 2. Selecting the release highlights
+Once you have cleaned up the changelog, the next step is to choose a few changes to highlight in the release post. These highlights usually focus on new features and enhancements, including performance and accessibility ones, but can also include important API changes or critical bug fixes.
 
-You should also include a performance audit at the end of the release post. You can use `bin/plugin/cli.js perf` to automatically do this.
+Given the big scope of Gutenberg and the high number of pull requests merged in each milestone, it is not uncommon to overlook impactful changes worth highlighting; it is recommended to make this step a collaborative effort and share your picks with the Gutenberg Core team.
 
-Compile this to a draft post on [make.wordpress.org/core](https://make.wordpress.org/core/); this post should be published after the actual release.
+#### 3. Drafting the release post
+
+When possible, the highlighted changes in the release post should include an animation (video or GIF) or a static image of them in use. Choosing between an animation or a static image will depend on the best way to showcase each highlight: while animations are better to demonstrate workflows, more direct highlights can be shown with an image; having too many video players adds a layer of friction for users to see the highlight, whereas too many simultaneous animated GIFs can distract and overwhelm the reader.
+
+These visual assets should maintain consistency with previous release posts; using lean, white themes helps in this regard and visually integrate well with the [make.wordpress.org/core](https://make.wordpress.org/core/) blog aesthetics. Including copyrighted material should be avoided, and browser plugins that can be seen in the browser canvas (spell checkers, form fillers, etc.) disabled when capturing the assets. If you would like help creating the visual assets, please ping somebody in the Gutenberg Core team in advance so that it can be planned for.
+
+#### 4. Running the performance tests
+
+Your post should also include a performance audit at the end, comparing the current Gutenberg release with both the previous one and the latest WordPress major version. You can use `bin/plugin/cli.js perf` to automatically do this for you by passing as parameters the tags to run the performance suite against, such as `bin/plugin/cli.js perf release/x.y release/x.z wp/a.b`
+
+#### 5. Publishing the post
+
+Compile this to a draft post on [make.wordpress.org/core](https://make.wordpress.org/core/); this post should be published after the actual release. Remember asking for peer review is encouraged by the [make/core posting guidelines](https://make.wordpress.org/core/handbook/best-practices/post-comment-guidelines/#peer-review)!
```

Comment:
Now that we also have volunteers from the community, there is a great chance that they don't have access to `make.wordpress.org/core` to create a draft and publish. 

Should we direct them to the process of seeking access through `#meta` or maybe it's better to prepare this and ping someone from the core team?